.github/workflows/continuous-integration-e2e.yml

```diff
@@ -16,13 +16,18 @@ jobs:
   e2e:
     name: '${{ matrix.browser }} - WP ${{ matrix.wp }}'
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental == true }}
     strategy:
       matrix:
         # TODO: add back Firefox once support is more mature.
         include:
           - browser: 'chrome'
             wp: 'latest'
             coverage: true
+          - browser: 'chrome'
+            wp: '5.7-RC2'
```

Comment:
We're already at RC 3.

Given that 5.7 stable is slated for today, we can probably just wait another day until "latest" becomes 5.7 automatically.